cmd/cosign/cli/policy_init.go

```diff
@@ -116,6 +114,7 @@ func initPolicy() *cobra.Command {
 			role.AddKeysWithThreshold(publicKeys, o.Threshold)
 			root.Roles["root"] = role
 			root.Namespace = o.ImageRef
+			root.PreviousRoot = "null"
```

Comment:
Can we just have it be an empty string, or omitted entirely? I'd rather not have the `"null"` string mean something special if we can avoid it.